signer/signer.go

```diff
@@ -34,7 +34,7 @@ import (
 )
 
 // signingAgent is the unprotected header field used by signature.
-const signingAgent = "Notation/1.0.0"
```

Comment:
@Two-Hearts Can you also send out an PR to the `main` branch as well as the `release-1.1` branch? We need to release 1.1.2. Also, we need to send out an PR to the `main` branch for `RELEASE_CHECKLIST.md`. Because of this field, the `notation-go` library requires a release process similar to `notation` CLI instead of `notation-core-go`.

Also, should this be `notation-go/1.2.0` instead?

Good thing is that it is for `GenericSigner`, which is mainly for testing purposes.

Reply:
I've already created an issue to track this: https://github.com/notaryproject/notation-go/issues/438. Will create PR to fix it later.

It impacts all plugins with `SIGNATURE_GENERATOR.RAW` capability, such as `AKV plugin`. When debugging, it is shown in the signing debug log.